apps/site/components/Containers/NavBar/index.module.css

```
@@ -86,6 +86,10 @@
     dark:border-neutral-900;
 }
 
+.searchButtonSkeleton {
+  @apply flex-grow;
+}
+
```

```suggestion
.searchButtonSkeleton {
  @apply my-px
    mr-2
    flex-grow
    rounded-xl;

  &:empty {
    @apply h-10;
  }
}
```

We can make the height and radius of this area a little more similar to the search box button

---

Ah, my bad 😅 The given height is not visible because the order of styles changed after the build, can you update it with a more specific class name?

Something like;
`.searchButtonSkeleton[data-inline-skeleton]` or `span.searchButtonSkeleton` should work

---

Nice catch, @canerakdas. I've amended and forced-pushed  my commit. Also tested with build - start. It looks good now. Thanks!